File-Formats-toc.html

```diff
@@ -121,9 +121,11 @@
             <li>
                 <a href="/file-formats/DocIO/Loading-and-Saving-document">Loading and Saving document</a>
 				<ul>
+					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core">ASP.NET Core</a></li>
 					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC">ASP.NET MVC</a></li>
 					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-ASP-NET">ASP.NET</a></li>
 					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-WPF">WPF</a></li>
+					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-Windows-Forms">Windows Forms</a></li>
```

Comment:
Can you please change the order of WPF and Windows Forms. Refer getting started page